### ISSUE TITLE
User profile page needed

### ISSUE THREAD

**Issue by singhayushh** [singhayushh/Bibleo#12]
Using simple html and css, I need a user profile page for this project, abiding the website's theme. 

Contributions are welcome.

**Comment by charlygg** [comment]
Hello, Can I try this issue?

**Comment by karkranikhil** [comment]
I would like to fix this.

**Comment by priyankaagrawal123** [comment]
I am interested. I can implement.
Please assign this issue to me.

**Comment by singhayushh** [comment]
I see three of you interested in this issue. 
Would you be able to discuss among yourself who to assign this issue to as I no preference except @charlygg showing up at first😅

You might go through the project and add any other pages that you might think is needed. All contributions are welcome

**Comment by Shweta200126** [comment]
I dont know where to comment so came right here ! I love your project ( the reason its about BOOKS ) and it looks beautiful. I tried but am new so dont know , is there any way to see the website as of rn ? if yes then pls link it!!

**Comment by singhayushh** [comment]
Hey @Shweta200126,
Thanks for your compliments and sorry for not stating clearly how to run the project locally.

I've deployed the project as it is currently, here - https://bibleo.herokuapp.com/

Many sections are incomplete so please don't mind 😅

**Comment by Shweta200126** [comment]
@singhayushh ITs just amazing website as of rn!!
I will love to be part of the final product whenever it launches.
Also the graphics are just so beautiful!! Who made them?? gotta follow ..... 

(Also I just recently joined so I have almost no clue how to run something locally !!)

**Comment by singhayushh** [comment]
I did the frontend designing myself. It still lacks important things like animations and responsiveness actually. 

I came up with the present theme after browsing [dribbble](https://dribbble.com) for hours. Here's the main shot where I got my inspiration from - https://dribbble.com/shots/10980760-Book-Library-page

And if you're taking about the images, they are from [undraw.co](https://undraw.co/) 

Thanks again for your compliments! 

(Also, no problem with that, you'll pick up soon. Nothing is too cumbersome. I've stepped into web development around 4-5 months ago)
